Wrappers/Python/test/test_algorithms.py

```diff
@@ -1261,20 +1261,16 @@ def test_compare_with_PDHG(self):
         tau = 1./normK
 
         pdhg = PDHG(f=F, g=G, operator=K, tau=tau, sigma=sigma,
-                    max_iteration = 100, update_objective_interval = 10)
+                    max_iteration = 500, update_objective_interval = 10)
```

Comment:
This test compares PDHG vs ADMM on a 64x64 `SHAPES` image.  Before had the original `SHAPES` image (300x200) and 100 iterations.  I believe now is even faster and more accurate since the image size is very small and both algs converged.